style.css

```diff
@@ -1,447 +0,0 @@
-    text-align: center;
-    cursor: pointer;
-    border-radius: 5px;
-}
-
-.section-three-content-input-btn {
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    align-items: center;
-}
-
-/* Footer */
-
-footer {
-    background-color: rgb(30, 30, 180);
-    color: white;
-    width: 100%;
-}
-
-.footer-container {
-    display: flex;
-    flex-direction: column;
-    padding: 3rem;
-}
-
-.footer-phone, .footer-mail,
-.footer-content {
-    margin-bottom: 1rem;;
-}
-
-.footer-container-img img {
-    width: 10rem;
-    margin-bottom: 3rem;
-    cursor: pointer;
-}
-
-.footer-phone, .footer-mail {
-    display: flex;
-    align-items: center;
-}
-
-.footer-phone {
-    margin-bottom: 2rem;
-}
-
-.footer-phone img, .footer-mail img {
-    width: 2rem;
-    cursor: pointer;
-}
-
-.footer-phone p, .footer-mail p {
-    margin: 0;
-    text-align: left;
-}
-
-.footer-mail {
-    margin-bottom: 3rem;
-}
-
-.footer-content-one, 
-.footer-content-two {
-    margin-bottom: 2rem;
-    line-height: 2rem;
-}
-
-.footer-content a {
-    text-decoration: none;
-    color: white;
-    font-size: 1.2rem;
-}
-
-.footer-content li {
-    cursor: pointer;
-} 
-
-footer li:hover {
-    font-size: 1.2rem;
-    color: bisque;
-}
-
-.footer-content a:hover {
-    font-size: 1.4rem;
-    color: coral;
-}
-
-.social-media ul {
-    display: flex;
-    justify-content: center;  
-}
-
-.social-media a:hover {
-    font-size: 4rem;
-}
-
-.social-media img {
-    width: 2rem;
-    padding: 0.6rem;
-    margin-left: 2rem;
-    border: 1px solid rgb(151, 92, 92);
-    border-radius: 50%;
-    background-color: white;
-    cursor: pointer;
-}
-
-footer .attribution a {
-    color: rgb(221, 221, 83);
-    text-decoration: none;
-}
-
-/* Responsive Rules */
-
-@media screen and (min-width: 900px) {
-    
-    /* Header */
-
-    .hamburger-icon {
-        display: none;
-    }
-
-    .menu {
-        display: flex;
-        width: 18rem;
-        margin-right: 1rem;
-    }
-    
-    .menu ul {
-        display: flex;
-        justify-content: space-around;
-        align-items: center;
-        align-items: center;
-        width: 100%;
-    }
-    
-    .menu a {
-        text-decoration: none;
-        font-size: 1.3rem;
-        cursor: pointer;
-    }
-
-    .menu a:hover {
-        font-size: 1.5rem;
-        color: coral;
-        transition: all 0.4s ease;
-    }
-
-    /* Main */
-
-    /* Section One */
-
-    .hero {
-        display: flex;
-        margin:2rem 3rem;
-    }
-
-    .hero-img {
-        order: 1;
-    }
-
-    .hero-content {
-        justify-content: center;
-    }
-
-    .hero-content h1 {
-        font-size: 1.5em;
-    }
-
-    .hero-content h1, p {
-        text-align: left;
-        text-indent: 0;
-    }
-
-    .hero-content-input-btn {
-        display: flex;
-        flex-direction: row;
-        justify-content: space-between;
-    }
-
-    .hero-content-btn {
-        width: 25%;
-    }
-    
-    #email {
```

Comment:
Don't use IDs in your CSS. You can use IDs for JavaScript, but when adding styles to HTML, target your elements using classes.